extensions/BMO/template/en/default/pages/anti-abuse.html.tmpl

```diff
@@ -0,0 +1,59 @@
+<!-- 1.0@bugzilla.org -->
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[% PROCESS global/header.html.tmpl 
+   title = "Bugzilla Anti-Abuse Policy" 
+   style = "li { margin: 5px } .heading { font-weight: bold }" %]
+
+<h2>Anti-Abuse Policy</h2>
+
+<p>
+  It is our intention that [% terms.Bugzilla %] remains a useful tool for reporting 
+  and commenting on [% terms.bugs %], feature requests and tasks for the Mozilla community.
```

Comment:
Also @Archaeopteryx is incorrect (understandably, because it's kind of weird); this should in fact be "remain", as it is a subjunctive (e.g. "I wish you were here", "I demand that you be on time", etc.).